data/tutorials/en/008_modules.md

```diff
@@ -140,7 +140,7 @@ val hello : unit -> unit
 
 (note the double asterisk at the beginning of the comment - it is a good habit
 to document .mli files using the format supported by
-[ocamldoc](/releases/{{! get LATEST_OCAML_VERSION_MAIN !}}/htmlman/ocamldoc.html))
+[ocamldoc](/releases/4.12.0/htmlman/ocamldoc.html))
```

Comment:
I'm guessing once we have releases we could point this to `/releases/latest` which redirects to the latest release we have?